hedera-node/hedera-mono-service/src/testFixtures/java/com/hedera/test/utils/KeyUtils.java

```diff
@@ -86,11 +89,31 @@ public static List<Key> sanityRestored(List<? extends HederaKey> jKeys) {
                 .toList();
     }
 
-    public static Key sanityRestored(HederaKey jKey) {
+    public static com.hederahashgraph.api.proto.java.Key sanityRestored(HederaKey jKey) {
         try {
             return JKey.mapJKey((JKey) jKey);
         } catch (Exception ignore) {
             throw new AssertionError("All keys should be mappable!");
         }
     }
+
+    public static com.hedera.hapi.node.base.Key sanityRestoredToPbj(HederaKey jKey) {
```

Comment:
Question: Why do we use HederaKey as param when it is directly cast as JKey?

Reply:
We are trying to remove all JKey types shortly and use HederaKey instead.